### PR TITLE
Fix data downscale paths

### PR DESCRIPTION
**Pull request by tancik**
This helps ignore dot files that might mess with processing.

### CODE REVIEW

File: nerfstudio/process_data/process_data_utils.py [hunk 1]
```diff
@@ -348,11 +347,8 @@ def downscale_images(
             assert isinstance(downscale_factor, int)
             downscale_dir = image_dir.parent / f"{folder_name}_{downscale_factor}"
             downscale_dir.mkdir(parents=True, exist_ok=True)
-            # Using %05d ffmpeg commands appears to be unreliable (skips images), so use scandir.
-            files = os.scandir(image_dir)
-            for f in files:
-                if f.is_dir():
-                    continue
+            # Using %05d ffmpeg commands appears to be unreliable (skips images).
+            for f in image_dir.glob("*.png"):
```
Comment (machenmusik) [inline nerfstudio/process_data/process_data_utils.py:351]: Are we quite sure that files to downscale will always be .png?

Comment (maturk) [reply nerfstudio/process_data/process_data_utils.py:351]: Could use something similar to: `allowed_exts = [".jpg", ".jpeg", ".png", ".tif", ".tiff"]`

**Comment by machenmusik** [comment]
LGTM

**Review by maturk — COMMENTED** [review]
LGTM :)

**Pull request merged**